tests/wpt/meta/html/semantics/forms/form-submission-0/form-submit-iframe-then-location-navigate.html.ini

```diff
@@ -0,0 +1,3 @@
+[form-submit-iframe-then-location-navigate.html]
+  [Verifies that location navigations take precedence when following form submissions.]
+    expected: FAIL
```

Comment:
Same here

Reply:
I'm going to leave these results in this commit. It's not worth the trouble to undo them.